t/run/runenv_hashseed.t

```diff
@@ -1,8 +1,23 @@
 #!./perl
 #
-# Tests for Perl run-time environment variable settings
+# Tests for PERL_PERMUTE_KEYS and PERL_HASH_SEED
 #
-# $PERL5OPT, $PERL5LIB, etc.
+# For PERL_PERMUTE_KEYS=NO or PERL_PERMUTE_KEYS=DETERMINISTIC we should
+# get the same key order each time.
+#
+# For PERL_PREMUTE_KEYS=RANDOM we should get a different key order each
+# time, or at least most times.
+#
+# For PERL_PERMUTE_KEYS=RANDOM the test is probabilistic and there is a
+# 1/131072 chance per repeat that we might get the same key order.
+#
+# For all the test modes there is also a chance that we might get the same
+# key order when we munge the hash seed, albeit a much lower chance.
+#
+# We do 50 repeats, so we have 50/131072 chance of getting the same
+# results for the probabilistic tests, which we round up to 1 since the
+# count of test fails can not be a fraction. So we allow 1 try per batch
+# of repeats to produce the same result before we fail the test.
```

Comment:
This is out of date now, we use `$allowed_fails`.

Reply:
Right, ill update the comment.